src/styles/bloks/section-title.scss

```diff
@@ -0,0 +1,26 @@
+.section-title {
+  margin-bottom: 48px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 100%;
```

Comment:
You are duplicating some styles for titles, try to avoid duplications especially if elements have the same logical meaning on the page